ddtrace/tracer/tracer.go

```diff
@@ -55,13 +63,21 @@ type tracer struct {
 
 	// pid of the process
 	pid string
+
+	// These integers track metrics about spans and traces as they are started,
+	// finished, and dropped
+	spansStarted, spansFinished, tracesDropped int64
 }
 
 const (
 	// flushInterval is the interval at which the payload contents will be flushed
 	// to the transport.
 	flushInterval = 2 * time.Second
 
+	// statsInterval is the interval at which span statistics will be sent with the
+	// statsd client.
+	statsInterval = 1 * time.Second
```

Comment:
You're right, but I think that can happen anyway because we can't fully sync with the datadog-agent, which internally creates these buckets too. I don't think it matters much though. For counts, if a bucket is empty it will just keep the previous value in the UI (not 0) so it will be fairly correct.